spec/fixtures/files/daemon.debian

```diff
@@ -0,0 +1,161 @@
+#! /bin/sh
```

Comment:
To me, having the entire body of both init scripts seems a bit like overkill, maybe?

Reply:
whats the point of checking the complete content of the init scripts/unit file anyway?

Reply:
Yeah, that's what I mean.

Reply:
the point is that it is hard to ensure correct file syntax with all those new lines and backslash. But your are the reviewer, tell me what you want.

Reply:
@sathieu i don't see how this check helps. if you change the files shipped in the module you need to change the  files shipped in the fixtures. so basically you are duplicating content without any benefit added

Reply:
I generally agree, but this kind of rspec helped for https://github.com/voxpupuli/puppet-prometheus/pull/48/files (by ensuring that those ```<%= @daemon_flags.join(" \\\n          ") %>``` or like were correct)

Reply:
how about testing it this way
```
          if ['debian-7-x86_64'].include?(os)
            # init_style = 'debian'

            it {
              is_expected.to contain_file('/etc/init.d/smurf_exporter').with(
                'mode'    => '0555',
                'owner'   => 'root',
                'group'   => 'root',
              )
              should contain_file('/etc/init.d/smurf_exporter').with_content(/DAEMON_ARGS=''\n/)
            }
```
also prometheus::daemon doesn't actually set $daemon_flags so the join in this context always evaluates to ''